designer/server/index.ts

```diff
@@ -1,9 +1,10 @@
 import { createServer } from "./createServer";
+const pino = require("pino")();
 
 createServer()
   .then((server) => server.start())
   .then(() => process.send && process.send("online"))
   .catch((err) => {
-    console.log(err);
+    pino.log(err);
```

Comment:
you can just use console.error here, as suggested by hapi-pino, https://github.com/pinojs/hapi-pino#usage